src/ui/dialogs/scenesettingsdialog.cpp

```diff
@@ -308,3 +310,26 @@ void SceneSettingsDialog::sNewSceneDialog(Document& document,
 
     dialog->show();
 }
+
```

Comment:
Some parts are not needed anymore, I recommend:
```
void SceneSettingsDialog::updateDuration(int index)
{
    const qreal fps = mFPSSpinBox->value();
    switch(index) {
        case 0: // Convert seconds to frames
            mMinFrameSpin->setValue(mMinFrameSpin->value() * fps);
            mMaxFrameSpin->setValue(mMaxFrameSpin->value() * fps);
            break;
        case 1: // Convert frames to seconds
            mMinFrameSpin->setValue(mMinFrameSpin->value() / fps);
            mMaxFrameSpin->setValue(mMaxFrameSpin->value() / fps);
            break;
        default:;
    }
}
```

Reply:
you should also probably qRound the value `setValue(qRound(whatever));`.

Reply:
yes, you are right. I took it in count before but now I dropped it...